src/main/java/com/team4/testingsystem/services/AnswerService.java

```diff
@@ -0,0 +1,14 @@
+package com.team4.testingsystem.services;
+
+import com.team4.testingsystem.dto.AnswerRequest;
+import com.team4.testingsystem.entities.Answer;
+
+public interface AnswerService {
+    Answer getById(long id);
+
```

Comment:
Maybe also add a method like `getAllByQuestion`? I think it's the most useful case.

Reply:
Not sure we will use it.
The use-cases here are:
1) get questions with their answers
2) save a question with its answers

So, what we actually need here is an entity, that will be used to map the data.

Reply:
Then shouldn't something like `@OneToMany` mapping be added to `Question`, so that we can get answers for a question without accessing the repository directly?

Reply:
Yes